{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:mod_authnz_external_project:mod_authnz_external:3.2.5:*:*:*:*:*:*:*",
	"cpeNameId": "BE8A528E-D5EA-457C-8239-9CFAAE81860F",
	"created": "2020-11-12T18:53:15.677",
	"lastModified": "2020-11-12T18:53:15.677",
	"titles": [
		{
			"title": "mod_authnz_external Project mod_authnz_external 3.2.5",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://exchange.xforce.ibmcloud.com/vulnerabilities/68799",
			"type": "Advisory"
		}
	]
}
